Packs/AHA/TestPlaybooks/AHA_TestPlaybook.yml

```
@@ -46,7 +46,7 @@ tasks:
 
         This automation runs using the default Limited User role, unless you explicitly change the permissions.
         For more information, see the section about permissions here:
-        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
+        For XSOAR 6 use the link https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations and for XSOAR 8 use the link https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script.
```

```suggestion
      - For Cortex XSOAR 6 use the link https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Administrator-Guide/Automations
       
         (Note: Replace 6.x with the version number of your environment, i.e., 6.12, 6.13, etc.)
     
       - For Cortex XSOAR 8 Cloud use the link https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script.
```
What about Cortex XSOAR on-prem? It should be added and should be something like this:
   - For Cortex XSOAR on-prem, use the link https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Scripts

   (Note: Replace 8.x with the version number of your environment, i.e., 8.6, 8.7, etc.)